### DISCUSSION TITLE
Spare PCBs available (one assembled)

### DISCUSSION THREAD

**Discussion by simontims** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#218] [General]
I have one assembled PCB for sale and 4 bare boards for free, in New Zealand.

Happy to ship anywhere though it might get expensive.

Say $15 USD for the assembled board one, plus shipping.
Bare boards free, shipping cost only, to give a bit back to the project which is been a lot of fun. One per person.

The assembled one has the headers for NodeMCU and level shifter and also two 6-pin JST-SM connection leads soldered on, and also includes a level shifter (note the NodeMCU is not included). It was working perfectly in my spa (new shape model S100103) - we sold the spa so have no use for the kit now.

![image](https://user-images.githubusercontent.com/19793185/156905778-c38ce2b0-9fbb-4728-b178-3bea6eb4aec2.png)

![image](https://user-images.githubusercontent.com/19793185/156905791-9e6e794b-a797-42c1-9f95-8288b93879c3.png)

![image](https://user-images.githubusercontent.com/19793185/156905816-44745a49-57cf-483b-9704-4dcdeb14b496.png)

**Comment by sdunnedeloitte** [comment]
Hey Simon, if the assembled board is still available I’d be very happy to pay for it and the shipping costs? Presumably from what I can see this should work with the Miami?

**Reply by simontims** [reply]
Sorry for the SLOW reply! Do you still want the assembled board? :)

**Reply by sdunnedeloitte** [reply]
Hey Simon, yes please! I'm in UK, let me know if you have available and be happy to buy one!

**Reply by sdunnedeloitte** [reply]
Just opened it up and checked and it's a 6-pin one model

![IMG_1662](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/52067517/c5a417f0-2b84-4ee9-aa32-1e38bc8605dd)

**Reply by simontims** [reply]
All good, did you want the assembled one or just the pcb? I have a relative visiting here who's going back to the UK in a couple of weeks so he can bring it back.

**Reply by simontims** [reply]
Oh, you already said assembled. OK, it's yours.

**Reply by simontims** [reply]
Drop me your email here and I'll get in touch.

**Comment by SchallertDK** [comment]
Hey Simon, i would like to by 2 of the empty boards :)

**Reply by billybigpotatoes** [reply]
Hi - any boards left?

**Reply by simontims** [reply]
Sorry for the slow replies! I still have the boards if anyone wants them.

**Reply by Remigiusz2020** [reply]
yep. I want to boards. How can I contact you?

**Reply by simontims** [reply]
Pop your email here and I'll contact you.

**Reply by sdunnedeloitte** [reply]
Amazing, thanks! Email is - sddunne at gmail dot com

**Comment by tunnus-1** [comment]
I also have some spare parts (not soldered, see below), could sell a kit for 15 eur + shipping cost. Two kits available (in Finland), if interested, reply with email address and we can then sort out the details. 

<img width="1537" alt="Screenshot 2023-06-21 at 9 56 29" src="https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/74300867/84961918-fe18-48f8-8311-20ff2afb11ba">

**Reply by rogerrally** [reply]
I am interested in one of the kits if you have any left. I am in the UK for postage calculations.

**Reply by Remigiusz2020** [reply]
Hi, can you  sell only V2b?

**Reply by rogerrally** [reply]
rogerfletcher@blueyonder.co.uk

**Reply by tunnus-1** [reply]
I only sell the complete kit (one still left, 4-wire)

**Reply by chumleychuff** [reply]
Hey tunnus-1, I'm interested in the full kit, I'm also in the UK, please excuse the disposable email address, if you mail me on it I'll mail back on my proper address.

Cbark@mailtothis.com

**Reply by tunnus-1** [reply]
All my boards / kits are now sold

**Comment by Remigiusz2020** [comment]
Hi. My e-mail ***@***.***   Drop me your email here and I&#39;ll get in touch. —  Reply to this email directly,  github.com view it on GitHub , or  github.com unsubscribe .  You are receiving this because you commented. Message ID:  &lt;visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/repo-discussions/218/comments/6275840 @ github . com&gt;

**Reply by simontims** [reply]
^ I don't think you can reply via email to provide your address. You may be able to post it here, or reveal it on your profile.

**Comment by henryfabu** [comment]
Is there any kit already available?
Thank you

**Reply by simontims** [reply]
I have bare boards.

**Comment by Remigiusz2020** [comment]
ok. fine. Is it v2b? 
     
      
       
        Dnia 27 czerwca 2023 21:06 Simon Tims ***@***.***&gt; napisał(a):
       
    
       
         
 I have bare boards.

**Reply by simontims** [reply]
There is no version on it, see pictures.
